.github/workflows/pkgcheck.yml

```diff
@@ -28,4 +28,9 @@ jobs:
         sudo ln -s /var/db/repos/gentoo/profiles/default/linux/amd64/17.1 /etc/portage/make.profile
     - name: Test with pkgcheck
       run: |
-        pkgcheck scan --exit --checks=-RedundantVersionCheck --keywords=-MissingAccountIdentifier
+        # There is a bug open with pkgcheck to fix the false positives with
```

Comment:
Worth linking an upstream bug report.

Reply:
I'll do so. I was a bit hesitant to reference it at all due to my previous spam issues with rebasing, but as this is a simple PR it shouldn't be an issue 

Or perhaps add a reference here in the PR?